<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaAgregar</class>
 <widget class="QDialog" name="VentanaAgregar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>VENTANA DE AGREGAR SERIE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>105</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de la serie: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>99</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Genero de la serie: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>105</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temporadas totales:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>138</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Episodios por temporada: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Estado: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>97</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Temporada actual:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Episodio actual: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVolver">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>81</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Nombre de la serie</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGenero">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>111</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Genero de la serie</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinTemTotal">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>141</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinEpTotal">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>168</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbEstado">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>195</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Seleccione una opcion</string>
   </property>
   <item>
    <property name="text">
     <string>Pendiente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Viendo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Acabada</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinTemActual">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>225</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinEpActual">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>252</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
